.gitignore

```diff
@@ -1,9 +1,12 @@
 .history/
 node_modules/
-pnpm-lock.yaml
+
 dist/
+doc/
+
 *.js.map
 *.js
+
 out.txt
 checked.txt
-doc/
+_test.ts
```

Comment:
Why ?

Reply:
I used _test for other things ill remove